integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -80,7 +81,7 @@ private void initGravitinoEnv() {
     gravitinoUri = String.format("http://127.0.0.1:%d", gravitinoPort);
   }
 
-  private void initHiveEnv() {
```

Comment:
I think hive env is ok, because it setup HDFS, HIVE related envriment.

Reply:
The jdbc catalog doesn't seem to require hive env, so I wanted to override the method in `SparkJdbcCatalogIT`, but the `initHiveEnv` method name is not appropriate for `SparkJdbcCatalogIT`. So I changed it to `initDatasourceEnv`

Reply:
fixed.